bcipy/task/paradigm/vep/stim_generation.py

```diff
@@ -0,0 +1,214 @@
+"""Functions related to stimuli generation for VEP tasks"""
+import itertools
+import math
+import random
+from typing import Any, List, Optional
+
+from bcipy.helpers.list import find_index, swapped
+from bcipy.helpers.stimuli import (InquirySchedule, get_fixation,
+                                   random_target_positions)
+
+
+def generate_vep_calibration_inquiries(alp: List[str],
```

Comment:
The stimuli module is getting pretty large and these functions seemed specific to this task. I also think they are more likely to change, so it made sense to me to split these out. I think it would be good to revisit the `stimuli` module and see if some of the things in there would make sense elsewhere. I'll create a ticket for that.